src/firebase/database.js

```diff
@@ -28,3 +28,15 @@ export const deleteGame = (roomKey) => {
   const gameRef = ref(database, `activeGames/${roomKey}`);
   return remove(gameRef);
 };
+
+export const doesGameExist = async (roomKey) => {
+  try {
+    const roomRef = ref(database, `activeGames/${roomKey}`);
+    const snapshot = await get(roomRef);
+    console.log(snapshot.exists());
+    return snapshot.exists();
+  } catch (error) {
+    console.error('Error checking room:', error);
+    throw error;
```

Comment:
Do you know what the throw here actually does?

Reply:
Passes the error to the file that calls it so it logs?

Reply:
Hmm so you're logging at both levels? I guess that's fine

Reply:
I can take it out as well

Reply:
Also I was being dumb, forgot this is just JavaScript and not firebase specific lol